packages/react/src/modules/slack/components/ConnectToSlackButton/ConnectToSlackButton.tsx

```diff
@@ -42,7 +99,7 @@ export const ConnectToSlackButton: React.FC<Props> = ({
   if (connectionStatus === "error") {
     return (
       <a
-        href={buildSlackAuthUrl()}
+        onClick={() => openSlackOauthPopup(buildSlackAuthUrl())}
         className="rnf-slackConnect-button rnf-slackConnect-button--error"
         onMouseEnter={() => setActionLabel("Reconnect")}
         onMouseLeave={() => setActionLabel("")}
```

Comment:
Semantically this should be a button now that you're using an onClick instead of an href